modules/rgbd/src/hash_tsdf.cpp

```diff
@@ -281,27 +348,27 @@ static inline Vec3i voxelToVolumeUnitIdx(const Vec3i& pt, const int vuRes)
     }
 }
 
-inline TsdfVoxel atVolumeUnit(const Vec3i& point, const Vec3i& volumeUnitIdx, VolumeUnitMap::const_iterator it,
-    VolumeUnitMap::const_iterator vend, int unitRes)
+TsdfVoxel HashTSDFVolumeCPU::atVolumeUnit(const Vec3i& point, const Vec3i& volumeUnitIdx, VolumeUnitIndexes::const_iterator it,
+    VolumeUnitIndexes::const_iterator vend, int unitRes) const
```

Comment:
If this function is not inlined (why?) then we don't need to pass `vend` and `unitRes` arguments.

Reply:
done